gremlin/README.md

```diff
@@ -105,6 +124,7 @@ Install the Helm chart
 ```shell
 helm install gremlin/gremlin \
     --name gremlin \
+    --namespace gremlin \
```

Comment:
these changes are not introduced as a result of chart changes, they were wrong on the current version of this README